001-cron.php

```diff
@@ -56,4 +56,14 @@ public function block_admin_page() {
 	}
 }
 
-new WPCOM_VIP_Cron_Control;
+// Allow testing of new approach to cron execution
+$whitelisted_sites = array();
+if ( VIP_GO_ENV && in_array( FILES_CLIENT_SITE_ID, $whitelisted_sites ) ) {
+	add_filter( 'wpcom_vip_go_enable_new_cron_control', '__return_true' );
```

Comment:
The filter's there to allow for local testing, as we set `VIP_GO_ENV` to false in https://github.com/Automattic/vip-go-mu-plugins/blob/cc326452d8745063ffef8032ab46de9b6ef7cc8e/000-vip-init.php#L14.